docs/can-guides/contribute/making-a-new-package.md

```diff
@@ -47,7 +47,7 @@ __3.__ List `canjs` in your `package.json`’s `keywords`.
 
 __4.__ Update the code to match the [guides/contributing/project-organization#Fileorganizationandresponsibilities File organization and responsibilities] section.  There are a few changes to make:
 
-- Change everything to CommonJS.  Use `require('module-name')` instead of `import 'module-name'`.
+- Change everything to CommonJS.  Useimport ` from 'module-name';` instead of `import 'module-name'`.
```

Comment:
Let's not change this file just yet. We don't have plans to change can-* source files to ES6 modules right now.